net-libs/rabbitmq-c/rabbitmq-c-0.13.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="RabbitMQ C client"
+HOMEPAGE="https://github.com/alanxz/rabbitmq-c"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/alanxz/${PN}.git"
+else
+	SRC_URI="https://github.com/alanxz/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~loong ppc ppc64 ~riscv ~s390 sparc x86"
```

Comment:
Even though this fixes a security bug, i think we need to bump this first only to testing - Especially since this PR basically also includes a version bump.